test/Sentry.Maui.Tests/MauiEventsBinderTests.CtMvvmAsyncRelayCommands.cs

```diff
@@ -0,0 +1,37 @@
+using CommunityToolkit.Mvvm.ComponentModel;
+using CommunityToolkit.Mvvm.Input;
+using Sentry.Maui.CommunityToolkit.Mvvm;
+
+namespace Sentry.Maui.Tests;
+
+public partial class MauiEventsBinderTests
+{
+    [Fact]
+    public async Task AsyncRelayCommand_AddsSpan()
+    {
+        var vm = new TestCtMvvmViewModel();
+        var button = new Button
+        {
+            Command = vm.TestCommand
+        };
+        var binder = new MauiCommunityToolkitMvvmEventsBinder(_fixture.Hub);
+        binder.Bind(button, _ => { });
+
+        button.Command.Execute(null!);
+
+        // we need to wait for async command to run
+        await Task.Delay(TimeSpan.FromSeconds(1.1));
+
+        // Assert
+        _fixture.Scope.Transaction.Should().NotBeNull("transaction should have been created");
```

Comment:
This test will always pass, even when `button.Command.Execute()` isn't called... since the transaction gets initialised to a null value in the Test Fixture.